stable/airflow/templates/configmap-env.yaml

```diff
@@ -22,6 +22,7 @@ data:
   DATABASE_PORT: "{{ .Values.externalDatabase.port }}"
   DATABASE_USER: "{{ .Values.externalDatabase.user }}"
   DATABASE_DB: "{{ .Values.externalDatabase.database }}"
+  DATABASE_PROPERTIES: "{{.Values.externalDatabase.properties}}"
   {{- end }}
```

Comment:
You will need to put a blank string under the `.Values.postgresql.enabled` case, otherwise you will get bash errors.

Example: `DATABASE_DB: ""`

Reply:
@thesuperzapper I don't see the bash errors in my local deployment and this field is only used in not .Values.postgresql.enabled scenario and no impact if postgresql enabled.
For the connection string template, do you mean the airflow.connection.alchemy and airflow.connection.celery.backend in _helpers.tpl? I already added this field for only .Values.postgresql.enabled = false scenario.

Reply:
As I said, I was mistaken, this is fine.